.gitattributes

```diff
@@ -10,6 +10,7 @@ tests/Test/AbstractTransformerTestCase.php export-ignore
 
 .appveyor.yml export-ignore
 .circleci/ export-ignore
+dev-tools/ export-ignore
```

Comment:
I'm neither for nor against this, but please do put it in the correct place alphabetically speaking (after `check_trailing_spaces.sh`).